pulpcore/cli/file/repository.py

```diff
@@ -69,10 +70,9 @@ def _content_callback(ctx: click.Context, param: click.Parameter, value: Any) ->
 CONTENT_LIST_SCHEMA = s.Schema([{"sha256": str, "relative_path": s.And(str, len)}])
 
 
-def _content_list_callback(ctx: click.Context, param: click.Parameter, value: Any) -> Any:
-    result = load_json_callback(ctx, param, value)
-    if result is None:
-        return None
+@load_file_wrapper
+def _content_list_callback(ctx: click.Context, param: click.Parameter, value: str) -> Any:
+    result = json_callback(ctx, param, value)
```

Comment:
Don't we still need the
```python
    if result is None:
        return None
```
part?

Reply:
Yes, we do! But, this is the right thing:

```python
    if not value:
        return value
```